docs/intro-to-oss/how-to-contribute-to-open-source.md

```diff
@@ -426,6 +445,12 @@ You'll need to have these tools donwloaded and installed on your local machine:
 
     :::
 
+### Completed PR Template
+
+Here's what a completed Pull Request Template for the guestbook looks like. Make sure yours looks like this, updated with your unique information.
+
+    ![Completed Pull Request Template](../_assets/images/completed-pr-template.png)
```

Comment:
Hi @pedaars,
Because this is not part of an (un)ordered list, can you please remove the indentation here? Thank you. 🙂 

```suggestion
![Completed Pull Request Template](../_assets/images/completed-pr-template.png)
```